contracts/libraries/PostLib.sol

```diff
@@ -331,15 +332,22 @@ library PostLib  {
             replyContainer.info.author,
             postContainer.info.communityId,
             UserLib.Action.EditItem,
-            UserLib.ActionRole.NONE,
+            isOfficialReply ? UserLib.ActionRole.CommunityModerator : 
```

Comment:
user without moderator rights can unmark official reply.
Will add unit test `giveCommunityModeratorPermission(user, ...) -> publication official reply -> revokeCommunityModeratorPermission(user, ...) ->  editReply (..., isOfficialReply false)`

Reply:
What is the resolution here? I think it is still an issue.

Reply:
Per discussion offline, if a user is no longer a moderator and edits a question that was official we need to uncheck the official flag automatically. Logic is correct.